jitsi-media-transform/src/main/kotlin/org/jitsi/nlj/PacketInfo.kt

```diff
@@ -140,6 +151,11 @@ open class PacketInfo @JvmOverloads constructor(
      */
     var payloadVerification = if (enablePayloadVerification) packet.payloadVerification else null
 
+    /**
+     * The type of the packet, used for tracking the sources of media being routed.
+     */
+    var packetType: PacketType = if (packet is RtcpPacket) PacketType.RTCP else PacketType.Misc
```

Comment:
This is initialized when the PacketInfo is created. I don't think the RTCP parser creates a new instance, so RTCP is only set for packets we generate?